client/web/src/search/results/StreamingSearchResults.tsx

```diff
@@ -90,12 +90,20 @@ export const StreamingSearchResults: FC<StreamingSearchResultsProps> = props =>
     const [allExpanded, setAllExpanded] = useState(false)
     const [showSavedSearchModal, setShowSavedSearchModal] = useState(false)
     const [showMobileSidebar, setShowMobileSidebar] = useState(false)
+    const [rankingEnabled, setRankingEnabled] = useTemporarySetting('search.ranking.experimental', false)
 
     // Derived state
     const trace = useMemo(() => new URLSearchParams(location.search).get('trace') ?? undefined, [location.search])
     const featureOverrides = useDeepMemo(
         // Nested use memo here is used for avoiding extra object calculation step on each render
-        useMemo(() => new URLSearchParams(location.search).getAll('feat') ?? [], [location.search])
+        useMemo(
+            () =>
+                [
+                    rankingEnabled ? 'search-ranking' : '-search-ranking',
+                    ...new URLSearchParams(location.search).getAll('feat'),
+                ] ?? [],
```

Comment:
I don't think this is needed anymore?
```suggestion
                ],
```

Reply:
Done.